src/tests/run_calc_for_tests.ts

```diff
@@ -28,13 +28,15 @@ Object.defineProperty(window, 'matchMedia', {
 });
 
 HTMLCanvasElement.prototype.getContext = () => ({
-  scale: () => {}
+  scale: () => {
+    // Mock function!
+  }
```

Comment:
```suggestion
  scale: () => {}
```
I recommend just adding `/* eslint-disable @typescript-eslint/no-empty-function */`